### ISSUE TITLE
Support for --filter options

### ISSUE THREAD

**Issue by ThomasAribart** [skyhookadventure/dynamodb-empty#1]
Hi and thanks for this useful package :)

I would need on my project to clear data in DynamoDB but only if a specific field matches a certain value. It would be nice to be able to provide filters, something like this:

```bash
dynamodb-empty --table tableName --filters '{ "myFieldName": { "$eq": "myFiedValue" } }'
```

**Comment by StanHannebelle** [comment]
Thanks for the package @alan-cooney !
I agree with @ThomasAribart. It would be great to have a filter option :)

**Comment by alan-cooney** [comment]
Makes sense! I don't actually use this much any more, but if you want to make a PR I'm happy to review it and push it live
